lib/better_html/tree/attribute.rb

```diff
@@ -0,0 +1,24 @@
+module BetterHtml
+  module Tree
+    class Attribute
+      attr_reader :node
+
+      def initialize(node)
+        @node = node
+      end
+
+      def loc
+        @node.name_parts.first&.location
+      end
+
+      def name
+        @node&.name&.downcase
+      end
+
+      def value
+        parts = @node.value_parts.reject{ |node| ["'", '"'].include?(node.text) }
```

Comment:
This isn't quite right; if the attribute is `foo="'"` then the value is rejected.

Reply:
good point!